<!-- Do not edit this file. It is automatically generated by API Documenter. -->

[Home](./index.md) &gt; [@twilio/voice-react-native-sdk](./voice-react-native-sdk.md) &gt; [TwilioErrors](./voice-react-native-sdk.twilioerrors_namespace.md) &gt; [GeneralErrors](./voice-react-native-sdk.twilioerrors_namespace.generalerrors_namespace.md) &gt; [CallCancelledError](./voice-react-native-sdk.twilioerrors_namespace.generalerrors_namespace.callcancellederror_class.md) &gt; [causes](./voice-react-native-sdk.twilioerrors_namespace.generalerrors_namespace.callcancellederror_class.causes_property.md)

## TwilioErrors.GeneralErrors.CallCancelledError.causes property

The incoming call was cancelled because it was not answered in time or it was accepted/rejected by another application instance registered with the same identity.

<b>Signature:</b>

```typescript
causes: string[];
```
